### DISCUSSION TITLE
Доски автоведения

### DISCUSSION THREAD

**Discussion by fixinit75** [metrostroi-repo/MetrostroiAddon#201] [Ideas]
Насколько я понял, они только прямые в метрострое, да ещё и длинные. В СПб они могли поворачивать, а в метрострое из-за этого нельзя полноценное автоведения на этих досках реализовать. Да и некрасиво получается, когда она стенку уходит.

**Comment by alexandr-sizov** [comment]
Имхо - лишнее. СБПП достаточно, доски нужны будут только если реализовывать #49.

**Comment by glebqip** [comment]
Вопрос, а как их поворачивать в кривых?

**Comment by fixinit75** [comment]
> Имхо - лишнее. СБПП достаточно, доски нужны будут только если реализовывать #49.

Почему же лишнее, если сейчас в метрострое автоведение от этих досок работает на составах с ПУАВ. В спб оно на ПУАВ и работает, помимо КСАУП.

**Comment by fixinit75** [comment]
> Вопрос, а как их поворачивать в кривых?

Можно сделать отдельные модели для этого, включить выбор в signaling tool.

**Comment by fixinit75** [comment]
[https://www.youtube.com/watch?v=1i45nA4l3QE](url)
Вот видео, где эти доски есть, программы начинаются с 1:42.
Если приглядеться, то видно, что они там состоят не из цельной доски, а из её частей, так можно в метрострое сделать.

**Comment by glebqip** [comment]
> Можно сделать отдельные модели для этого, включить выбор в signaling tool.

А если нам нужно захватить кусок кривой? Это довольно тяжело получится всё собирать по длине, плюс кривые могут быть любого радиуса всё же Не уверен что тут прям 100% вариант получится, чтобы было удобно

**Comment by alexandr-sizov** [comment]
> Почему же лишнее, если сейчас в метрострое автоведение от этих досок работает на составах с ПУАВ. В спб оно на ПУАВ и работает, помимо КСАУП.

Сейчас возможна работа и на досках и на СБПП, приоритетным является СБПП, и для метростроя это наиболее подходящий вариант. Повторюсь, по моему мнению, реализовывать гнущиеся доски надо только в том случае если делать КСАУП, по которому на сколько я знаю практически нет рефов. Я думаю на данный момент из досок надо оставить только дверные программы, а все остальные убрать, ибо смысла в них нет.

**Comment by fixinit75** [comment]
> А если нам нужно захватить кусок кривой? Это довольно тяжело получится всё собирать по длине, плюс кривые могут быть любого радиуса всё же Не уверен что тут прям 100% вариант получится, чтобы было удобно

Можно попробовать, как это выглядит на видео. Там видно, что используются несколько этих досок, а не одна цельная.

**Comment by fixinit75** [comment]
> Я думаю на данный момент из досок надо оставить только дверные программы, а все остальные убрать, ибо смысла в них нет.

Ну, с этим я частично соглашусь.

**Comment by glebqip** [comment]
> Можно попробовать, как это выглядит на видео. Там видно, что используются несколько этих досок, а не одна цельная.

Как в кривых ты всё равно их расположишь

**Comment by cyber-dream** [comment]
Спецмодельки?) Динамические сплайны на луа?))
